src/main/groovy/org/shipkit/gradle/exec/ShipkitExecTask.java

```diff
@@ -0,0 +1,39 @@
+package org.shipkit.gradle.exec;
+
+import org.gradle.api.DefaultTask;
+import org.gradle.api.tasks.TaskAction;
+import org.shipkit.internal.gradle.exec.ShipkitExec;
+
+import java.util.Collection;
+import java.util.LinkedList;
+
+/**
+ * Generic Gradle task that allows composing multiple executable command line invocations.
+ * Very useful for scenarios where we cannot easily use multiple Gradle Exec tasks.
+ */
+public class ShipkitExecTask extends DefaultTask {
```

Comment:
On a second thought, let's discuss first :) The reason I changed the name from Composite -> Shipkit was because:
 - the task has more responsibilities than composing exec commands (better reporting, prefixing output)
 - we will use this task instead of plain Gradle ExecTask everywhere. This way, we get the benefits of clean output. In most cases, this task would not be a composite, it would be a singular command.

Thoughts?